plugins/generic/htmlArticleGalley/HtmlArticleGalleyPlugin.inc.php

```diff
@@ -100,6 +100,8 @@ function articleDownloadCallback($hookName, $args) {
 
 		if ($galley && $galley->getFileType() == 'text/html' && $galley->getFileId() == $fileId) {
 			echo $this->_getHtmlContents($request, $galley);
+			$returner = true;
+			HookRegistry::call('FileManager::downloadFileFinished', array(&$returner));
```

Comment:
Rather than calling the File Manager's hook here, I'd suggest calling a different (new) hook, and registering against it in UsageEventPlugin.